### DISCUSSION TITLE
Side slip angle issue

### DISCUSSION THREAD

**Discussion by woojmn** [DLR-VSDC/IEEE-MVC-2023#23] [Q&A]
Hello,

I have a question about the side slip angle.

![error_picture](https://user-images.githubusercontent.com/95162423/216268807-cd9371b4-0440-421e-a335-1bd138fa4828.PNG)

As you can see the picture, in the cycle #4, the steer and yaw rate are almost zero but the side slip angle is large value. 

I think it is weird, so I wonder your idea for this issue.

**Comment by tobolar** [comment]
@woojmn Please check https://github.com/DLR-VSDC/IEEE-MVC-2023/issues/24#issuecomment-1419406532
